inc/components/post-excerpt/component.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Post excerpt.
+ *
+ * Get the post excerpt.
+ *
+ * @package Irving_Components
+ */
+
+namespace WP_Irving;
+
+use WP_Irving\Component;
+
+if ( ! function_exists( '\WP_Irving\get_registry' ) ) {
+	return;
+}
+
+/**
+ * Register the component and callback.
+ */
+get_registry()->register_component_from_config(
+	__DIR__ . '/component',
+	[
+		'callback' => function( Component $component ): Component {
+
+			// Get the post ID from a context provider, or fallback to the global.
+			$post_id = $component->get_config( 'post_id' );
+			if ( 0 === $post_id ) {
+				$post_id = get_the_ID();
+			}
+
+			return $component->set_config( 'content', html_entity_decode( get_the_excerpt( $post_id ) ) );
```

Comment:
We should probably do `apply_filters( 'the_excerpt', get_the_excerpt() );` so this is treated like `the_excerpt()` in WordPress core.

Reply:
Didn't realize it didn't run through the filters using `get_the_excerpt()`. Good catch. Thanks.